test_export.py

```
@@ -0,0 +1,161 @@
+            total_linhas=45,
+            comandos_maiusculas=3,
+            pontos_exclamacao=2,
+            caracteres_invisiveis=2
+        ),
+        recomendacoes=[
+            "⚠️ RISCO ALTO: Revisão manual obrigatória antes de processar",
+            "Aplicar sanitização nos comandos detectados",
+            "Verificar origem do documento",
+            "Investigar padrões de caracteres invisíveis"
+        ],
+        timestamp=datetime.utcnow(),
```

Ruff (0.14.5)

`datetime.datetime.utcnow()` used

(DTZ003)